public/index.html

```diff
@@ -1,15 +1,11 @@
 <!DOCTYPE html>
 <html>
 
-<head>
-  <title>My test page</title>
-</head>
-
 <body>
   <div id='overview'></div>
 </body>
 <script type="module" src='../dist/overview.js'></script>
 <script crossorigin src="https://unpkg.com/react@17/umd/react.development.js"></script>
```

Comment:
Your code is running fine and I'm not sure if this is a real convention or not but I generally but scripts requiring cdn's inside the <head> rather than below the <body>

Reply:
inside, which? I just put them on the page where I've seen others?